lib/kitchen/directions/bake_chapter_introductions/v2.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeChapterIntroductions
+  class V2
+    def bake(book:, chapter_objectives_strategy:)
+      book.chapters.each do |chapter|
+        introduction_page = chapter.introduction_page
+
+        introduction_page.search("div[data-type='description']").trash
+        introduction_page.search("div[data-type='abstract']").trash
+
+        title = introduction_page.title.cut
+        title.name = 'h2'
+        Kitchen::Directions::MoveTitleTextIntoSpan.v1(title: title)
+
+        chapter_objectives_html =
+          Kitchen::Directions::BakeChapterIntroductions.bake_chapter_objectives(
+            chapter: chapter,
+            chapter_objectives_strategy: chapter_objectives_strategy
+          )
+
+        intro_content = introduction_page.search('[data-type="note"].chapter-objectives').cut
```

Comment:
As I write in my other comment I think there is a mistake in old recipe. Probably this part is not necessery. Note should be modified like it is done for `chapter_objectives_html`.

Reply:
Do we have a decision from Larissa about that?